lib/virt_autotest/utils.pm

```diff
@@ -231,15 +235,10 @@ sub create_guest {
         my ($autoyastURL, $diskformat, $virtinstall);
         $autoyastURL = $autoyast;
         $diskformat = get_var("VIRT_QEMU_DISK_FORMAT") // "qcow2";
-
-        assert_script_run "qemu-img create -f $diskformat /var/lib/libvirt/images/xen/$name.$diskformat 20G", 180;
-        assert_script_run "sync", 180;
-        script_run "qemu-img info /var/lib/libvirt/images/xen/$name.$diskformat";
-
         $extra_args = "$linuxrc autoyast=$autoyastURL $extra_args";
         $extra_args = trim($extra_args);
         $virtinstall = "virt-install $v_type $guest->{osinfo} --name $name --vcpus=$vcpus,maxvcpus=$maxvcpus --memory=$memory,maxmemory=$maxmemory --vnc";
-        $virtinstall .= " --disk /var/lib/libvirt/images/xen/$name.$diskformat --noautoconsole";
+        $virtinstall .= " --disk path=/var/lib/libvirt/images/$name.$diskformat,size=20,format=$diskformat --noautoconsole";
```

Comment:
Do you have any test cases which use "diskformat=raw"?
It may need more time to create a raw dev than qcow2 one.

Reply:
will test it with raw format.

Reply:
make test: VIRT_QEMU_DISK_FORMAT=raw. It works well.